concourse/pipelines/templates/external-table-tpl.yml

```
@@ -31,7 +31,7 @@ resources:
 {% endif %}
     tag: latest
     username: _json_key
-    password: ((data-gpdb-ud-google-json-key))
+    password: ((ud/pxf/secrets/google-project-id-ccp-ci-service-account-key))
```

```suggestion
    password: ((ud/pxf/secrets/pxf-cloudbuild-service-account-key))
```

Since this cred is being used to fetch a container image, we should use the CloudBuild service account key.